tests/pytorch_tests/test_pytorch_quantization_wrapper.py

```diff
@@ -20,6 +20,8 @@
 
 from mct_quantizers.pytorch.quantize_wrapper import PytorchQuantizationWrapper
 
+WEIGHT = 'weight'
```

Comment:
where are you using it? if its a constant that is used not only in the tests than take it from the relevant constants file